### DISCUSSION TITLE
[Projects Beta] Unable to add Draft item to empty view

### DISCUSSION THREAD

**Discussion by jahredhope** [community/community#6871] [Projects and Issues] [answered]
This might be a new bug or perhaps we've never got a view down to zero, but I've noticed the "Add Item" column that appears at the bottom of lists on the List View doesn't appear if there are no items in the view.

This seems like a pretty prominent bug, so I'm thinking I might be missing something.

# Expected 
A view that has at least one item in it: contains a row saying "Add Item":

![Screen Shot 2021-10-28 at 1 23 11 pm](https://user-images.githubusercontent.com/13903378/139175458-c4681ec4-bfc0-466a-be18-e3bf2e59b3c2.png)

# Actual
A view that has no items in it: contains no such row:

![Screen Shot 2021-10-28 at 1 22 46 pm](https://user-images.githubusercontent.com/13903378/139175465-40f84308-147f-40bd-96c5-faa0a11c3e83.png)

**Comment by azenMatt** [comment, marked as answer]
@jahredhope - do you happen to have a group by applied to this view as well? 

I can re-create this in the instance where there are no items and a column has a group by. 

This was because we hide empty groups... and missed this case. We are getting a fix out for that right now.

**Reply by jahredhope** [reply]
Just tested now, yes I can only re-create with a Group By. Thanks.

I remember adding a Group By to try to fix the issue, but perhaps I had a Group By in the first place as well. I'll remove the Group By now as a work around and look forward for a fix for Group By.

Thanks

**Reply by azenMatt** [reply]
I'm glad that's the case! Ok the fix should be live for this by the end of the week
